modules/organization/src/endpoints/mod.rs

```diff
@@ -0,0 +1,75 @@
+use crate::service::OrganizationService;
+use actix_web::{get, web, HttpResponse, Responder};
+use trustify_common::db::query::Query;
+use trustify_common::db::Database;
+use trustify_common::model::Paginated;
+use utoipa::OpenApi;
+
+pub fn configure(config: &mut web::ServiceConfig, db: Database) {
+    let service = OrganizationService::new(db);
+    config
+        .app_data(web::Data::new(service))
+        .service(web::scope("/api/v1/organization").service(all).service(get));
+}
+
+#[derive(OpenApi)]
+#[openapi(
+    paths(all, get),
+    components(schemas(
+        trustify_model::organization::OrganizationHead,
+        trustify_model::organization::OrganizationSummary,
+        trustify_model::organization::OrganizationDetails,
+        trustify_model::organization::PaginatedOrganizationSummary,
+    )),
+    tags()
+)]
+pub struct ApiDoc;
+
+#[utoipa::path(
+    tag = "organization",
+    context_path = "/api/v1/organization",
```

Comment:
...and remove all the `context_path` attributes in the `utoipa::path`'s...

Reply:
If we remove the context_path from utoipa, then the swagger-ui/openapi stuff shows *the wrong path*, since it's ignorant of the web::scope, seemingly?